README.md

```diff
@@ -16,8 +16,50 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using `.animate()` method when possible](#avoid-using-`.animate()`-method-when-possible)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Avoid using `.animate()` method when possible
+### Why?
+Most of the time the same can be accomplished with toggling classes and using CSS transition and/or keyframes.
```

Comment:
I guess animation is also 'styling' and therefore is the responsibility of CSS. So it's not so much about that it can be done in CSS, but that it *should* be done in CSS.